CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ### Changed
 
 - The `SpanModels` function is now exported from the `go.opentelemetry.io/otel/exporters/zipkin` package to convert OpenTelemetry spans into Zipkin model spans. (#2027)
+- Replaced usages of the `trace.Link` type in the SDK package with an equivalent `Link` struct that also counts the number of dropped link's attributes. (#2118)
```

Comment:
Can we rewrite this from the user perspective? Likely this needs to be transformed into an item in the `Add` section describing the new type used.

Reply:
Maybe I've stretched it too far, but from a user/usage perspective, I can think in something like:

> Added a new `Link` type under the SDK `otel/sdk/trace` package that counts the number of attributes that were dropped for surpassing the `AttributePerLinkCountLimit` configured in the Span's `SpanLimits`. This new type replaces the equal-named API `Link` type found in the `otel/trace` package for most usages within the SDK. For example, instances of this type are now returned by the `Links()` function of  `ReadOnlySpan`s provided in places like the `OnEnd` function of `SpanProcessor` implementations.

But don't hesitate on suggesting a better/shorter alternative if you had something in mind!

Reply:
LGTM